user/test.py

```diff
@@ -52,3 +56,21 @@ def test_delete_trip(self):
         user = User.objects.first()
         self.assertEqual(user.locations.count(), 0)
         self.assertEqual(result.status_code, 204)
+
+    def test_unpublished_articles_does_not_appears_if_the_author_not_logged(
+            self):
+        Article(title='<h1>Article that must not appear<br></h1>',
+                content='<p>content</p>', author=self.user,
+                is_published=False).save()
+        result = self.client.get('/@emailtest')
+        self.assertNotIn(b'Article That Must Not Appear', result.data)
+        self.assertEqual(result.status_code, 200)
+
+    def test_unpublished_articles_does_appears_if_the_author_is_logged(self):
+        login_user(self)
+        Article(title='<h1>Article that must appear<br></h1>',
```

Comment:
no html please, it must be cleaned out anyways (and you have a dedicated test for that) ;)